contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,96 @@
+
+To switch between projects, navigate to project settings, or to create new projects, use the project switcher in the middle of the top bar. You can also quickly go to the current project's settings from the sidebar.
+
+> **Note:** Multiple projects within an organization belong to our premium team-oriented offering. To use this feature, [set up PostHog Cloud billing](https://app.posthog.com/organization/billing) or [contact us for a self-hosted license](/pricing).
+
+### Ways of organizing projects
+
+There are several practical ways of using projects:
+
+- A project per environment – you may want to test out analytics instrumentation in development or staging environments, while keeping that test data separate from production.
+- A project per product – if you have several unlinked products, it makes sense to reflect that separation in PostHog too.
+- A project per multiple products or the whole company – if your offering is interlinked or you want to track the user journey holistically (e.g. how many company blog readers convert to paid product users), you can use a single project for all digital assets.
+
+Which way is the best fit for you depends on the characteristics of your product and business.
+
+## Permissions
+
+By default all organization members have access to all its projects, at their organization-wide access level.
+This makes for the smoothest collaboration. However, if you require access to data to be more granular, see [Project-based permissioning](#project-based-permissioning) below.
+
+### Access levels
+
+There are three access levels in PostHog, each with more privileges than the one below it. There can be any number of members at each level, except Owner, of which there is always one in the organization.
+
+See permissions at each level below:
+
+| | Member (base level) | Administrator (elevated level) | Owner (unique top level) |
+| --- | --- | --- | --- |
+| Viewing and querying project data | ✅ | ✅ | ✅ |
+| Leaving the organization | ✅ | ✅ | ✅ |
+| Inviting new members | ✅ | ✅ | ✅ |
+| Billing management | ✅ | ✅ | ✅ |
+| Project creation and deletion | ❌ | ✅ | ✅ |
+| Project settings management (incl. [project-specific membership](#project-based-permissioning)) | ❌ | ✅ | ✅ |
+| Organization settings management (incl. membership) | ❌ | ✅ | ✅ |
+| Organization deletion | ❌ | ❌ | ✅ |
+
+Access levels can be viewed and changed in the Members section of organization settings.
+
+In projects with [project-based permissioning](#project-based-permissioning), there's also a Members section in project settings, which pertains to project-specific memberships.
+
+### Project-based permissioning
+
+If you'd like to restrict access to data within the organization to only those who need it, you can use PostHog's project-based permissioning.
+
+**Enable project-based permissioning** is a switch in project settings that can be turned on (or off) at any time by organization-wide Administrators and the Owner.
```

Comment:
```suggestion
Any Administrator or Owner can make a project private or open at any time.
```

Reply:
I'm suggesting here from a user standpoint to refer to this setting as having "Private" or "Open" projects, instead of project-based permissioning.